CompStoreAPI.postman_collection

```diff
@@ -403,7 +521,139 @@
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://localhost:8080/pcs",
+							"protocol": "http",
+							"host": [
+								"localhost"
+							],
+							"port": "8080",
+							"path": [
+								"pcs"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "create 201 Auth Prod",
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJtMmJKODFRak5pcG84emU4bXUzVG9KamstaUozNmVYdHJVOGJjX0p4TjE0In0.eyJleHAiOjE3MTAxMTY3NDYsImlhdCI6MTcxMDExNjQ0NiwianRpIjoiYjI4MGVkYzYtZWEyOC00ODdhLThhMDgtYWNlMjkwNWJmOWQxIiwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo4MDkwL3JlYWxtcy9jb21wc3RvcmUiLCJhdWQiOiJhY2NvdW50Iiwic3ViIjoiYWVkOGRkNDktNjA4ZS00Mzg3LWIxMzItMTJlZWVhZTA5NmI1IiwidHlwIjoiQmVhcmVyIiwiYXpwIjoiY29tcHN0b3JlLWFwaSIsInNlc3Npb25fc3RhdGUiOiI3MDgzYTJjYS0zNWE3LTQ1YTktODM3MC0yNzRjMGE3MjA0MjQiLCJhY3IiOiIxIiwiYWxsb3dlZC1vcmlnaW5zIjpbIi8qIl0sInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJvZmZsaW5lX2FjY2VzcyIsImFkbWluIiwidW1hX2F1dGhvcml6YXRpb24iLCJkZWZhdWx0LXJvbGVzLWNvbXBzdG9yZSJdfSwicmVzb3VyY2VfYWNjZXNzIjp7ImNvbXBzdG9yZS1hcGkiOnsicm9sZXMiOlsiYWRtaW4iXX0sImFjY291bnQiOnsicm9sZXMiOlsibWFuYWdlLWFjY291bnQiLCJtYW5hZ2UtYWNjb3VudC1saW5rcyIsInZpZXctcHJvZmlsZSJdfX0sInNjb3BlIjoiZW1haWwgcHJvZmlsZSIsInNpZCI6IjcwODNhMmNhLTM1YTctNDVhOS04MzcwLTI3NGMwYTcyMDQyNCIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJhZG1pbiJ9.DhFBMw2-yqYHqS3HzD1XnQ2EgPkNGaX03C7tFOZR6TQDmpYdPWWefHcUEraveMqrnIiSxeQU7A_s9l2Gn1_m6tpDn2kKXECheoQyErVCcnrk0dp4XxINyYvBwTkSqROzRlo-LwhE9V9toHjQEQP5NYvapwpD5hpehK_-yCFZBViNDrgVs08Y-kRMjm_N7PWH6qko0CVTmnrspugIdco2fqhYbMVBayRTxth4sKeoPf7M2VKfiJR-JHBrDVXaovnVX9Jrq-64BL1z7Rn_iiDlhJcydyvv1d2OK0VnXV7cNqRXA3NeoYq8nUV_uPp05tq_tHg96IXkJ78icknwc7DA_w",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"processorBrand\": \"45e8716b-0720-4d2a-82ca-8c4f155d1e7b\",\r\n    \"processorName\": \"New processor name\",\r\n    \"graphicsCardBrand\": \"b6496490-c055-4cac-a1c3-06e4d1a402cb\",\r\n    \"graphicsCardName\": \"New graphics card name\",\r\n    \"ramCapacity\": \"32 GB\",\r\n    \"driveCapacity\": \"1 TB\",\r\n    \"driveType\": \"HDD\",\r\n    \"operatingSystem\": \"5c96a57f-b8f7-4e66-92e1-85290ce745fe\",\r\n    \"price\": 1999.99\r\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "http://localhost:8080/pcs",
+							"protocol": "http",
+							"host": [
+								"localhost"
+							],
+							"port": "8080",
+							"path": [
+								"pcs"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "create 201 operating system null",
```

Comment:
Endpointy z zadania z walidacją. Musiałem je tu dodać mimo, że to nie to zadanie, bo wszystko jest w jednym pliku